.travis.yml

```diff
@@ -34,6 +34,12 @@ jobs:
         - pip install -r requirements.txt
         - sls plugin install -n serverless-python-requirements
         - sls deploy
+        - pip install awscli
```

Comment:
why install awscli with pip, instead of npm?

Reply:
That was Amazon's recommended install method: https://docs.aws.amazon.com/cli/latest/userguide/installing.html

If npm works, then groovy.